Client.go

```diff
@@ -377,7 +377,14 @@ func (c *Client) authenticated() bool {
 func (c *Client) Authenticate() error {
 	// We cannot use Client.do() in this function, since this function can be
 	// called from do() upon session renewal.
-	URL := "https://sso.garmin.com/sso/signin?service=https%3A%2F%2Fconnect.garmin.com%2Fmodern%2F&gauthHost=https%3A%2F%2Fconnect.garmin.com%2Fmodern%2F&generateExtraServiceTicket=true"
+	URL := "https://sso.garmin.com/sso/signin?service=https%3A%2F%2Fconnect.garmin.com%2Fmodern%2F" +
+		"&gauthHost=https%3A%2F%2Fconnect.garmin.com%2Fmodern%2F" +
+		"&generateExtraServiceTicket=true" +
+		"&generateTwoExtraServiceTickets=true" +
+		"&useCustomHeader=false" +
+		"&mfaRequired=false" +
+		"&rememberMyBrowserShown=false" +
+		"&rememberMyBrowserChecked=false"
```

Comment:
`useCustomHeader`, `mfaRequired`, `rememberMyBrowserShown` and  `rememberMyBrowserChecked` are unneeded.